### DISCUSSION TITLE
Seamless Tiles

### DISCUSSION THREAD

**Discussion by 3Diva** [lllyasviel/Fooocus#2759] [Ideas]
Midjourney has the ability to create seamless tiles, is there any plans to make seamless tiling available in Fooocus?

**Comment by muneeb227** [comment]
Being a 3d artist, I am looking to generate seamless textures (diffuse, normal, roughness, AO). Is it possible with Fooocus?

**Reply by IPv6** [reply]
nope, it`s not a focus of Fooocus - so no special features/treatment for such task

**Reply by mashb1t** [reply]
No, not OOTB. There is https://pypi.org/project/img2texture/ which basically multiplies an image x4 and smoothes the edges.
